.github/workflows/php.yml

```diff
@@ -0,0 +1,73 @@
+name: CI FOSHttpCacheBundle
+
+env:
+  SYMFONY_PHPUNIT_DIR: "$HOME/symfony-bridge/.phpunit"
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - php-version: '7.2'
+            symfony-version: ''
+            stability: ''
+          - php-version: '7.3'
+            symfony-version: ''
+            stability: ''
+          - php-version: '7.2'
+            symfony-version: ''
+            stability: 'dev'
+          - php-version: '7.3'
+            symfony-version: ''
+            stability: 'dev'
+          - php-version: '7.2'
+            symfony-version: '^4'
+            stability: ''
+          - php-version: '7.3'
+            symfony-version: '^4'
+            stability: ''
+          - php-version: '7.4'
+            symfony-version: '^4'
+            stability: ''
+          - php-version: '8.0'
+            symfony-version: '^4'
+            stability: ''
+          - php-version: '7.3'
+            symfony-version: '^5'
+            stability: ''
+          - php-version: '7.4'
+            symfony-version: '^5'
+            stability: ''
+          - php-version: '8.0'
+            symfony-version: '^5'
+            stability: ''
+    name: PHP ${{ matrix.php-version }} Test on Symfony ${{ matrix.symfony-version }}
+    steps:
+      - name: Pull the code
+        uses: actions/checkout@v2
+      - name: Install PHP and Composer
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php-version }}
+          tools: composer:v2
+      - name: Check PHP Version
+        run: php -v
+      - name: Stability
+        run: composer config minimum-stability ${{ matrix.stability }}
+        if: ${{ matrix.stability }}
+      - name: Symfony version
+        run: composer require --no-update symfony/flex && composer config extra.symfony.require ${{ matrix.symfony-version}}
+        if: ${{ matrix.symfony-version }}
+      - name: Composer update
+        run: composer update --prefer-stable --prefer-lowest --prefer-dist --no-interaction
```

Comment:
Did you intend to always prefer lowest? There’s no conditional here.

Reply:
Hi @ddeboer !
Yes, to avoid multiply test suites... do you think it's a bad practice and i should also test the latests ? 
Thanks for your review :)

Reply:
Thank you for your PR! 😄

In my opinion, tests should also run against the latest versions of vendors, so having `--prefer-lowest` on all tests seems wrong. @dbu What do you think?

Reply:
i agree with @ddeboer . sorry if i was unclear with https://github.com/FriendsOfSymfony/FOSHttpCacheBundle/pull/554#discussion_r542225232

the idea of prefer lowest is to have 1 build with the lowest supported php version that installs the oldest allowed combination of packages, to check that the minimum requirements we have in composer.json work as advertised. (e.g. we could be using a class that has only been added in 1.3 of a dependency, but in composer.json we say `^1.2`. if we only build with the newest versions, we don't notice that mistake in the CI).
the rest of the builds however should use the newest version of dependencies, as that is the most likely way how the bundle will be running. 

i suggest to use the composer_flag variable to manage these options, but only set the flag in a matrix.include that uses the lowest php version we support.